crates/feldera-types/src/program_schema.rs

```diff
@@ -342,70 +341,50 @@ pub enum IntervalUnit {
 }
 
 /// The available SQL types as specified in `CREATE` statements.
-#[derive(Serialize, ToSchema, Debug, Eq, PartialEq, Clone, Copy)]
+#[derive(ToSchema, Debug, Eq, PartialEq, Clone, Copy)]
 #[cfg_attr(feature = "testing", derive(proptest_derive::Arbitrary))]
 pub enum SqlType {
     /// SQL `BOOLEAN` type.
-    #[serde(rename = "BOOLEAN")]
```

Comment:
is removing these part of the fix?

Reply:
Not really, but once there is not longer a derive of `Serialize` and/or `Deserialize`, annotations are no longer possible (nor meaningful, as (de-)serialization is now a custom function).

Reply:
They'll throw an error:
```
error: cannot find attribute `serde` in this scope
```

Reply:
I meant more like if making them a different casing is part of the fix

Reply:
No, deserialization failed because default serialization generated an object instead of a string for the interval types.